cdap-data-fabric/pom.xml

```diff
@@ -207,6 +207,11 @@
       <artifactId>otj-pg-embedded</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
```

Comment:
why does this need to be added? Was it a transitive that is no longer transitive anymore?

Reply:
twill was bringing kafka for scala 2.10. I excluded it as it's not compatible with 2.11/2.12 and added 2.11 explicitly when needed.